odml/tools/odmlparser.py

```diff
@@ -61,11 +62,18 @@ def write_file(self, odml_document, filename):
         msg = ""
         for err in validation.errors:
             if err.is_error:
-                msg += "\n\t- %s %s: %s" % (err.obj, err.rank, err.msg)
+                # msg += "\n\t- %s %s: %s" % (err.obj, err.rank, err.msg)
+                msg += "\n- %s" % err
         if msg != "":
             msg = "Resolve document validation errors before saving %s" % msg
             raise ParserException(msg)
 
+        report = validation.report()
+        if report:
+            msg += "The saved Document contains formal issues."
```

Comment:
Is "formal" an appropriate term here?  Not sure.

Reply:
Any suggestions what to use instead?